core/src/main/java/com/datastax/oss/driver/internal/core/config/typesafe/TypesafeDriverExecutionProfile.java

```diff
@@ -269,7 +270,7 @@ public DriverExecutionProfile without(@NonNull DriverOption option) {
   @NonNull
   @Override
   public Object getComparisonKey(@NonNull DriverOption option) {
-    // No need to cache this, it's only used for policy initialization
+    // Override the default implementation because we can do much simpler
```

Comment:
nit: maybe could you expand on that comment? When I am reading code it does not give me a lot of info. I think that there is too little context here.

Reply:
The implementation here is a one-liner, vs 8 lines in the default impl, that's what the comment is referring to.

Reply:
👍 I'll rephrase the comment.